src/App.tsx

```diff
@@ -465,6 +465,9 @@ function AppContent() {
             <Route exact path={APP_PATHS.NURSERY_WITHDRAWALS}>
               <NurseryWithdrawals reloadTracking={reloadTracking} />
             </Route>
+            <Route exact path={APP_PATHS.NURSERY_WITHDRAWALS_V2}>
```

Comment:
we shouldn't need this,, just remove the `exact` from line 465 and keep the same path name without v2?

Reply:
I was getting trouble with the nursery withdrawals details, they have same path, that's why I created this new one

Reply:
got it, some suggestions, will let you decide

1) will be cleanest to handle all withdrawals routes inside of withdrawals/index.tsx (we don't need to do everything in App.tsx.. maybe just the root level non-exact path handling)

2)  add tracking v2 checks for the other routes
{ !trackingV2 && (
  <Route v1 withdrawals detail route >
)}

3) for v2 withdrawals, add /withdrawals/:siteId/:detailsId and handle that route to go to details component

also i just realized for v2 we need to rename the `< Withdrawal Log` link to `Withdrawals`

long term - the decision to use `/withdrawals2/id` for planting site and `/withdrawals/id`  for details may not look good in retrospect.
a one time effort may be worth it, thoughts?

Reply:
> may need to check if BE has any withdrawals links in its emails/notifications, i can do that

Excellent, thank you!! 



> another suggestion: `/withdrawals/site/<siteId>/details/<detailId>` for v2 `/withdrawals/<detailid>` for v1
> 
> this might work easier
> 
> if you add a route check for `/withdrawals/site/...` before the `/withdrawals/<detailsid>` things will work cleanly
> 
> Do all route checking in withdrawals/index.tsx have App.tsx route to withdrawals/index.tsx for non-exact path match of `/withdrawals`

Ok, will do this

Reply:
Thanks @constanzauanini , i think we will be happier with that approach. if you want, we can get this in and do the routing separately once I see if the BE has any dependencies. also will be a more isolated PR and easier to focus on.